Documentation/artifact-format.md

```diff
@@ -232,8 +411,41 @@ or multiple files listed.  For example:
 { "files" : ["core-image-minimal-201608110900.ext4", "core-image-base-201608110900.ext4"]}
 ```
 
+### type-info (version 3)
+
+Format: JSON
 
-### type-info
+A file that provides information about the type of package contained within the
+tar file. The first and the only required entry is the type of the update
+corresponding to the type in `header-info` file.
+It can also contain some additional parameters extending or modifying the global
+`artifact_provides` set of parameters specific for a given update type.
+
+```
+{
+  "type": "rootfs-image"
+  "artifact_provides": [
+          { "provides_type": ["provides_value", "provides_value"] },
```

Comment:
Maybe some more entries here as well, depending on what we decide to put in each header file.

Reply:
I've added `rootfs_image_checksum`.